specification/hybridconnectivity/resource-manager/Microsoft.HybridConnectivity/stable/2023-03-15/hybridconnectivity.json

```diff
@@ -0,0 +1,1093 @@
+          ],
+          "x-ms-enum": {
+            "name": "result",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ServiceConfigurationResourcePatch": {
+      "description": "The service details under serviceconfiguration for the target resource.",
+      "type": "object",
+      "properties": {
+        "systemData": {
+          "description": "System data of endpoint resource",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "x-ms-client-flatten": true
+        },
+        "properties": {
+          "description": "The service configuration properties.",
+          "$ref": "#/definitions/ServiceConfigurationPropertiesPatch",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ServiceConfigurationPropertiesPatch": {
+      "type": "object",
+      "description": "Service configuration details",
+      "properties": {
+        "port": {
+          "type": "string",
+          "pattern": "^[0-9]+$",
+          "description": "The port on which service is enabled."
+        }
+      }
+    },
+    "EndpointsList": {
+      "description": "The list of endpoints.",
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of endpoints list.",
+          "type": "string"
+        },
+        "value": {
+          "description": "The list of endpoint.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EndpointResource"
+          }
+        }
+      }
+    },
+    "ManagedProxyResource": {
+      "type": "object",
+      "description": "Managed Proxy",
+      "required": [
+        "proxy",
+        "expiresOn"
+      ],
+      "properties": {
+        "proxy": {
+          "type": "string",
+          "description": "The short lived proxy name."
+        },
+        "expiresOn": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The expiration time of short lived proxy name in unix epoch."
+        }
+      }
+    },
+    "ManagedProxyRequest": {
+      "type": "object",
+      "description": "Represent ManageProxy Request object.",
+      "required": [
+        "service"
+      ],
+      "properties": {
+        "service": {
+          "type": "string",
+          "description": "The name of the service."
+        },
+        "hostname": {
+          "type": "string",
+          "description": "The target host name."
+        },
+        "serviceName": {
+          "type": "string",
+          "description": "The name of the service.",
+          "enum": [
+            "SSH",
+            "WAC"
+          ],
+          "x-ms-enum": {
+            "name": "serviceName",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ListCredentialsRequest": {
+      "type": "object",
+      "description": "Represent ListCredentials Request object.",
```

Comment:
>"Represent ListCredentials Request object." [](http://example.com/codeflow?start=21&length=43)

Try to avoid using mostly meaningless words like 'represent' and 'object' are here, in descriptions